<launch>
<!-- -->
  <arg name="camera_frame"  default="camera" />
  <!-- 
    <node pkg="tf2_ros" exec="static_transform_publisher" name="base2map" args="0 0 0 0 0 0 map base" />
    <node pkg="tf2_ros" exec="static_transform_publisher" name="camera2base" args="0 0 1 0 0 -1.95 base $(var camera_frame)" />
  -->
  
  
  <!-- 
  <include file="$(find-pkg-share aruco_detect)/launch/aruco_detect.launch.xml"> 
    <arg name="camera_frame"  value="$(var camera_frame)"/>
  </include>
  -->
  
  <node pkg="marker_slam"  exec="marker_slam"  output="screen" >
    <param name="map_frame"  value="map" />
    <param name="base_frame"  value="base" />
    <param name="camera_frame"  value="$(var camera_frame)" />
    <param name="estimate_frame"  value="base" />
    <param name="verbose"  value="false" />
    <param name="map_file"  value="$(find-pkg-share marker_slam)/launch/map.txt" />
    <param name="landmark"  value="50, 51, 52, 53, 54, 55" value-sep=", " />
  </node>
<!-- 
-->

</launch>